### DISCUSSION TITLE
User deleted task from project, how to return it?

### DISCUSSION THREAD

**Discussion by kurkijar** [web2project/web2project#343] [Q&A] [answered]
User deleted a task from project. I can find the task using task-id url. In there there are only time markings, but it cant be edited.
In db i cant find the task anymore from the tasks table, the corresponding task id is empty and trying to search using task name returns nothing.
How can i return the deleted task to the project?

**Comment by caseysoftware** [comment, marked as answer]
You can't. Once you delete something, it is deleted.
